client/signup/steps/user-type/form.jsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Card from 'components/card';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import { recordTracksEvent } from 'state/analytics/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class UserTypeForm extends Component {
+	static propTypes = {
+		userType: PropTypes.string,
+		submitForm: PropTypes.func.isRequired,
+
+		// from localize() HoC
+		translate: PropTypes.func.isRequired,
+	};
+
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			userType: props.userType,
+		};
+	}
+
+	handleRadioChange = event => this.setState( { userType: event.currentTarget.value } );
+
+	handleSubmit = event => {
+		const { userType } = this.state;
+
+		event.preventDefault();
+
+		this.props.recordTracksEvent( 'calypso_signup_actions_submit_user_type', {
+			value: userType,
+		} );
+
+		this.props.submitForm( userType );
+	};
+
+	renderRadioOptions() {
+		const { translate } = this.props;
+
+		const userTypeQuestions = [
+			{
+				slug: 'creator',
+				label: translate( 'Myself' ),
+			},
+			{
+				slug: 'builder',
+				label: translate( 'Someone else' ),
+			},
+		];
+		return userTypeQuestions.map( userTypeProperties => (
```

Comment:
Seems like we're defining this function on every re-render. Move out in a separate method?